joint_angles:
    vertical_pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #intermediate_pick_pose: [179.0, 17.0, 182.0, 231.0, 0.0, 56.0, 92.0]

    #values are in degrees
    perceive_table: [-11.865325927734354, 21.10895347595215, 129.39122009277344, -52.20489501953125, 20.05187225341797, -110.82797241210935, 38.08707809448242]
    #intermediate_place_pose_right: [220.6 , 340.0 , 344.8 , 285.5 , 4.6 , 271.9 , 115.1]
    #place_pose_right: [219.9 , 328.3 , 347.2 , 260.1 , 7.1 , 308.7 , 112.2]
    perceive_left_pose: [209.7 , 279.9 , 91.9 , 250.3 , 12.6 , 332.66 , 49.88]
    #intermediate_place_pose_left: [163.2 , 325.5 , 344.7 , 308.3 , 8.1 , 264.6 , 57.9]
    #place_pose_left: [160.9 , 315.3 , 347.2 , 282.1 , 9.9 , 300.7 , 53.2]
    #demo_pose: [215.070602417, 355.983215332, 327.985229492, 278.106719971, 4.49846887589, 354.735626221, 88.8878555298]
    probe_place_pre_holder_pose: [34.885589599609375, 61.007503509521484, -179.64237976074222, -114.3497619628906, 39.43632125854492, 84.97713470458986, 88.7705078125]
    probe_place_in_holder_pose: [60.870471954345696, 86.61841583251953, 143.26763916015625, -107.55418395996097, 49.08712387084961, 67.82467651367188, 125.28645324707031]
    byod_safe_pose: [-2.5110778808593577, 32.06026077270508, 176.4428253173828, -116.52104187011717, 85.53815460205078, -93.52474975585933, -56.978363037109375]
    probe_place_in_holder_pose: [50.20372772216797, 71.61961364746094, 159.5046844482422, -110.62054443359372, 45.41108703613282, 74.9588851928711, 106.2127456665039, 0.2010382610634847]
    probe_place_after_probing: [-13.977508544921903, 24.39418601989746, -139.77947998046875, -128.1422882080078, -49.84103393554687, -14.257019042968757, -54.07586669921872, 41.54553613615975]
    safe_place_for_re_picking_above_01: [3.2176926136016846, 16.899463653564453, 177.47506713867188, -141.6487579345703, -10.381378173828121, 74.80049133300781, 89.61261749267578, 0.2010382610634847]
    safe_place_for_re_pikcing_middle_02: [45.60859298706055, 60.76933288574218, 179.28521728515625, -128.56182861328128, 47.95956039428711, 95.177001953125, 98.31245422363281, 0.2010382610634847]
    multimeter_pre_pose: [87.67285919189453, -20.001098632812518, 179.99795532226562, -146.00729370117188, -0.0009155273437315431, -50.00091552734378, 90.0]
    safe_pose_after_probe_placement: [2.03061842918396, 36.78822326660156, -147.75141906738284, -84.50717163085938, -20.154083251953168, -62.33142089843749, 128.91610717773438]
    probe_safe_after_pluck: [22.855220794677734, 33.05693817138672, 173.20434570312503, -132.2525177001953, 22.18228530883789, 73.19831848144531, 89.84786987304689]
    perceive_board_pose: [-10.840637207031197, 0.17449672520160675, 178.48037719726565, -87.32131958007812, -0.011993408203142748, -92.52655029296876, 118.89358520507812, 0.2019237814421938]
